tmt/steps/prepare/ansible.py

```diff
@@ -70,8 +70,11 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
 
     .. warning::
 
-        When specifying playbooks with paths, all paths must be
-        relative to the metadata tree root.
+        When specifying playbooks with paths:
+
+        * If a metadata tree root exists, all paths must be relative to the metadata tree root.
+        * If the metadata tree root does not exist, \
+all paths must be relative to the current working directory.
```

Comment:
ReST supports breaking lines, add new-line & align the new line with the beginning of the line above:

```
        * If the metadata tree root does not exist,
          all paths must be relative to the current working directory.
```

Reply:
Seems to be covered.